libfwupdplugin/fu-dpaux-device.c

```diff
@@ -72,7 +74,20 @@ fu_dpaux_device_probe(FuDevice *device, GError **error)
 		    g_path_get_basename(fu_udev_device_get_sysfs_path(FU_UDEV_DEVICE(device)));
 		fu_device_set_logical_id(device, logical_id);
 	}
-	return fu_udev_device_set_physical_id(FU_UDEV_DEVICE(device), "pci,drm_dp_aux_dev", error);
+
+	if (!fu_udev_device_set_physical_id(FU_UDEV_DEVICE(device), "pci,drm_dp_aux_dev", error))
+		return FALSE;
+
+	/* only populated on real system, test suite won't have udev_device set */
+	tmp = fu_udev_device_get_sysfs_attr(FU_UDEV_DEVICE(device), "name", NULL);
+	if (tmp != NULL)
+		fu_device_set_name(device, tmp);
+
+	/* FuUdevDevice->probe */
+	if (!FU_DEVICE_CLASS(fu_dpaux_device_parent_class)->probe(device, error))
+		return FALSE;
```

Comment:
in general, incorporate() should be good enough to copy over the baseclass. Is there a reason we have to set the logical and physical IDs before calling the baseclass probe? This feels like it's working around a deeper bug, for instance something missing from `fu_udev_device_incorporate()`

Reply:
No; there isn't a strong reason for the call order.  I'll move it to the beginning of the function.